### ISSUE TITLE
Can we extract text from word page by page

### ISSUE THREAD

**Issue by Anupam750** [KevM/tikaondotnet#129]
please explain, How can i get text from pdf, word page by page?

**Comment by KevM** [comment]
Look at the unit tests. 🤗

**Comment by bugybunny** [comment]
I‘m also interested in this (but for PDFs). I guess you mean      tikaondotnet/src/TikaOnDotNet.Tests/text_extraction.cs, right? I couldn‘t find an example for that.

**Comment by Anupam750** [comment]
I have also checked it and did not found anything related to page in code as you said..

**Comment by KevM** [comment]
Not sure I follow what you are trying to do. Can you tell me more detail about what you need?

There are examples of text extraction against many file types in the tests.

Kevin Miller
@kevm

**Comment by bugybunny** [comment]
We want to extract the text of a .pdf or .docx (or .doc dunno, I can’t speak for Anupam75) page by page. I currently just use `new TextExtractor().Extract(filename.pdf) .Text` to get the text from a PDF. But I would love to know where the page breaks are.

**Comment by KevM** [comment]
I don't believe that is an option that Tika offers. Tika is a high level abstration. If you wanted to do that more precisely I would look at using POI directly. https://poi.apache.org/text-extraction.html

**Comment by bugybunny** [comment]
Can you make something from this answer? https://stackoverflow.com/a/6271696/4040068

I thought that it’s not possible and I know that Tika is just an abstraction layer to get the content from so many different formats. I have to use .NET (so PDFBox itself is not an option, there’s a PDFBox with IKVM but it’s not maintained anymore) and TikaOnDotNet was the only pdf extraction tool that I can use concerning the license which is free. So I was hoping I can still use it for a bit more than just plain extraction, but it’s not really a problem :)

**Comment by KevM** [comment]
You might be able to hook into Tika to get the raw markup. Not sure. Let me know what you find out.
